controller-server/src/main/java/com/yahoo/vespa/hosted/controller/permits/CloudAccessControl.java

```diff
@@ -33,7 +34,12 @@ public CloudTenant createTenant(TenantPermit permit, List<Tenant> existing, List
     }
 
     @Override
-    public void deleteTenant(TenantPermit permit, Tenant tenant, List<Application> applications) {
+    public Tenant updateTenant(TenantPermit tenantPermit, List<Tenant> existing, List<Application> applications) {
+        throw new NotSupportedException("Update is not supported here, as it would entail changing the tenant name.");
```

Comment:
Use `UnsupportedOperationException` from standard library instead of pulling in JAX-RS things.

Reply:
Of course >_< Thanks!